vendor/github.com/knative/test-infra/scripts/README.md

```diff
@@ -134,7 +134,7 @@ This is a helper script for Knative E2E test scripts. To use it:
 1. Write logic for the end-to-end tests. Run all go tests using `go_test_e2e()`
    (or `report_go_test()` if you need a more fine-grained control) and call
    `fail_test()` or `success()` if any of them failed. The environment variables
-   `DOCKER_REPO_OVERRIDE`, `K8S_CLUSTER_OVERRIDE` and `K8S_USER_OVERRIDE` will be
+   `KO_DOCKER_REPO`, `K8S_CLUSTER_OVERRIDE` and `K8S_USER_OVERRIDE` will be
```

Comment:
You've edited vendored files here. You'll need to throw those out of the diff to this repo.

Reply:
Done